generators/app/templates/src/StarterKit/StarterKit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <VersionPrefix>1.0.0-BUILDNUMBER</VersionPrefix>
+    <VersionPrefix>5.0.0-BUILDNUMBER</VersionPrefix>
```

Comment:
Deze mocht op 1 blijven staan (dit is de versie van de gegenereerde applicatie, 5 is de versie van de generator)

Reply:
Sorry, ik dacht dat ik erbij had gezet dat enkel package.json aangepast moest worden, blijkbaar vergeten.